### DISCUSSION TITLE
Public, private and protected in a class

### DISCUSSION THREAD

**Discussion by raimondoDiSangro** [gaphor/gaphor#1547] [Q&A] [answered]
Hello,
is it possible to change the access modifiers of class?
Looks like I can only write "+" public modifiers.

Thank you!

**Comment by h8672** [comment, marked as answer]
According to wikipedia there's some that tell which access level are markers for class attributes.
```bash
https://en.wikipedia.org/wiki/Class_diagram#Class-level_relationships
+ Public
- Private
# Protected Seemed to work
~ Package <- This didn't work when I tested.
```
You just type the access level (+ - #) in front of attribute when adding them to list.

**Reply by raimondoDiSangro** [reply]
Many thanks, that's exactly how it works, but honestly I found it a bit cumbersome at first!
